retrievalmarket/impl/integration_test.go

```diff
@@ -79,7 +79,12 @@ func requireSetupTestClientAndProvider(bgCtx context.Context, t *testing.T, payC
 	retrievalmarket.RetrievalProvider) {
 	testData := tut.NewLibp2pTestData(bgCtx, t)
 	nw1 := rmnet.NewFromLibp2pHost(testData.Host1)
-	rcNode1 := testnodes.NewTestRetrievalClientNode(testnodes.TestRetrievalClientNodeParams{PayCh: payChAddr})
+	cids := tut.GenerateCids(2)
+	rcNode1 := testnodes.NewTestRetrievalClientNode(testnodes.TestRetrievalClientNodeParams{
+		PayCh:          payChAddr,
+		CreatePaychCID: cids[0],
```

Comment:
Write an integration test case for the add funds flow. We need to know if it actually works.